File: Proyecto_APK_Movil/app/src/main/java/com/example/proyecto_apk/MisLibros/LibroModel.java
package com.example.proyecto_apk.MisLibros;

import java.util.Calendar;

public class LibroModel {
  String id;
  String titulo;
  String autor;
  String editorial;
  int cantidadPaginas;
  Calendar fecha;
  String genero;
  String descripcion;
  String imagenPortada;

  LibroModel(){}

  public String getId() {
    return id;
  }

  public void setId(String id) {
    this.id = id;
  }

  public String getTitulo() {
    return titulo;
  }

  public void setTitulo(String titulo) {
    this.titulo = titulo;
  }

  public String getAutor() {
    return autor;
  }

  public void setAutor(String autor) {
    this.autor = autor;
  }

  public String getEditorial() {
    return editorial;
  }

  public void setEditorial(String editorial) {
    this.editorial = editorial;
  }

  public int getCantidadPaginas() {
    return cantidadPaginas;
  }

  public void setCantidadPaginas(int cantidadPaginas) {
    this.cantidadPaginas = cantidadPaginas;
  }

  public Calendar getFecha() {
    return fecha;
  }

  public void setFecha(Calendar fecha) {
    this.fecha = fecha;
  }

  public String getGenero() {
    return genero;
  }

  public void setGenero(String genero) {
    this.genero = genero;
  }

  public String getDescripcion() {
    return descripcion;
  }

  public void setDescripcion(String descripcion) {
    this.descripcion = descripcion;
  }

  public String getImagenPortada() {
    return imagenPortada;
  }

  public void setImagenPortada(String imagenPortada) {
    this.imagenPortada = imagenPortada;
  }
}

File: PlataformaVirtualHumanos/src/app/_services/paciente_services/agenda_services/MisLibrosServices/LibroModel.ts
export class LibroModel
{
    id:any;
    titulo:any;
    autor:any;
    editorial:any;
    cantidadPaginas:any;
    fecha:any;
    genero:any;
    descripcion:any;
    imagenPortada:any;
    constructor(){
    }
    set_id(id:any){
        this.id=id;
    }
    get_id(){
        return this.id
    }
    set_titulo(titulo:any){
        this.titulo=titulo;
    }
    get_titulo(){
        return this.titulo
    }
    set_autor(autor:any){
        this.autor=autor;
    }
    get_autor(){
        return this.autor
    }
    set_editorial(editorial:any){
        this.editorial=editorial;
    }
    get_editorial(){
        return this.editorial
    }
    set_cantidadPaginas(cantidadPaginas:any){
        this.cantidadPaginas=cantidadPaginas;
    }
    get_cantidadPaginas(){
        return this.cantidadPaginas
    }
    set_fecha(fecha:any){
        this.fecha=fecha;
    }
    get_fecha(){
        return this.fecha
    }
    set_genero(genero:any){
        this.genero=genero;
    }
    get_genero(){
        return this.genero
    }
    set_descripcion(descripcion:any){
        this.descripcion=descripcion;
    }
    get_descripcion(){
        return this.descripcion
    }
    set_imagenPortada(imagenPortada:any){
        this.imagenPortada=imagenPortada;
    }
    get_imagenPortada(){
        return this.imagenPortada
    }
}